docs/source/operation/backup.md

```diff
@@ -250,189 +233,110 @@ There are multiple methods for backing up the Gluu Server. A few recommended str
-                echo "---" >> gluu_secrets.yaml && \
-                kubectl get secret gluu-jackrabbit-postgres-pass -n gluu -o yaml >> gluu_secrets.yaml
-                ```        
-        
-            
-            1. If this is a restore it likely means the helm deployment of Gluu is currupt. Delete the helm deployment of Gluu in preperation for a new fresh one:
-            
-                ```bash
-                helm delete <release-name> -n <gluu-namespace>
-                ```
-            
-            1. Create `gluu` `Secret` and `ConfigMap` from backup saved previosuly:
-            
-               ```bash
-               kubectl create -f gluu_main_secret.yaml && kubectl create -f gluu_main_cm.yaml
-               ```
-            
-            1. Run the install command for helm:
-            
-                ```bash
-                helm install <release-name> -f ./values.yaml . -n <gluu-namespace>
-                ```
-                
-            1. Move the backup ldifs to the new opendj pod at `/opt/opendj/ldif`
-            
-            1.  Choose the backup of choice and run the command below:
-            
-                ```bash
-                /opt/opendj/bin/import-ldif --hostname localhost --port 4444 --bindDN "cn=Directory manager" --backendID userRoot --trustAll --ldifFile /opt/opendj/ldif/backup-4.ldif --bindPassword "<Password>"
-                ```
-            
-            1. Preform a rolling update of each service, forexample :             
-            
-               ```bash
-               kubectl rollout restart gluu-deployment -n gluu
-               ```
+            /opt/opendj/bin/import-ldif --hostname localhost --port 4444 --bindDN "cn=Directory manager" --backendID userRoot --trustAll --ldifFile /opt/opendj/ldif/backup-1.ldif --bindPassword "<Password>"
+            ```                     
             
+    === "SQL"
+
+        SQL database backup is a common feature whether in a User-managed or Cloud-Managed deployment.
+
+    ## Configmaps and Secrets - Backup and Restore    
+
+    ### Backup of Configmaps and Secrets    
+
+    1.  Configmaps backup:
+    ```bash
+    kubectl get configmap gluu -n <namespace> -o yaml > configmap-backup.yaml
+    ```
+
+    2.  Secrets backup:
+    ```bash
+    kubectl get secret gluu -n <namespace> -o yaml > secret-backup.yaml
+    ```
+
+    3.  Get the user supplied values:
+
+        Save the values.yaml that was used in the initial `gluu` installation using helm.
+
+        In the event that the user supplied or override values yaml was lost, you can obtain it by executing the following command:
+        ```bash
+        helm get values <release name> -n <namespace>
+        ```
+
+    4.  Keep note of installed chart version:
+    ```bash
+    helm list -n <namespace>
+    ```
+
+    Keep note of the chart version. For example: `backup-chart-version`
+
+    ### Restore of Configmaps and Secrets 
+
```

Comment:
You need to specify here that the persistence (database) needs to be restored first